aodntools/timeseries_products/hourly_timeseries.py

```diff
@@ -540,7 +519,7 @@ def hourly_aggregator(files_to_aggregate, site_code, qcflags, input_dir='', outp
     nc_aggregated = nc_aggregated.drop('OBSERVATION')
 
     ## add global attributes
-    contributor_name, contributor_email, contributor_role = get_contributors(files_to_aggregate, input_dir=input_dir)
+    contributor_name, contributor_email, contributor_role = utils.get_contributors(files_to_aggregate, input_dir=input_dir)
```

Comment:
wrap that beast to return the add_attribute instead. ```get_contributor_attr```

Reply:
Done.